packages/p2p/src/types/my-ads.types.ts

```diff
@@ -0,0 +1,85 @@
+export type TPaymentMethodOrder = {
```

Comment:
yes.. this would work .. but i think it would be `Partial<Record<'bank_transfer' | 'other' | 'wechat_pay' | 'alipay' | 'skrill' | 'paypal' ,number>>` since they are optional